src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,61 @@
+.calendar {
+  max-width: 7 * $cell-size + 7 * $cell-gap + 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+  margin: 50px auto;
+
+  $days:
+    "mon" 1,
+    "tue" 2,
+    "wed" 3,
+    "thu" 4,
+    "fri" 5,
+    "sat" 6,
+    "sun" 7;
+
+  @each $day, $dayNum in $days {
+    &__start-day--#{$day} {
+      &__cell:nth-child(1) {
+        margin-left: $cell-size*($dayNum - 1) + $cell-gap * $dayNum;
+      }
+    }
+
+  }
+
+  @for $i from 3 through 1 {
+    &__month-length--#{31 - $i} {
+      &__cell:nth-last-child(-n + #{$i}) {
+        display: none;
+      }
+    }
+  }
+
+  &__cell {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    @include square($cell-size);
+    border: 1px solid #000;
+    background-color: #eee;
+
+    @for $i from 1 through 31 {
+      &--#{$i}::before {
+        content: "#{$i}";
+        position: absolute;
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: $font-size;
+        text-align: center;
+        left: $base-shift;
+        top: $base-shift;
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translateY(-20px);
+      transition: 0.5s;
```

Comment:
don't use it like that, because it's the same as transition: all 0.5s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md
![image](https://user-images.githubusercontent.com/91411692/222665179-6078c1fa-ec4d-4048-96e6-068e3f8b8981.png)

```suggestion
```